codecov.yml

```diff
@@ -0,0 +1,6 @@
+comment: off
+coverage:
+  status:
+    project:
+      default:
+        target: auto
```

Comment:
I say we go with something we know we're ok with until we solve the latching problem.